src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/IChatClient.cs

```diff
@@ -33,6 +33,11 @@ public interface IChatClient : IDisposable
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>The response messages generated by the client.</returns>
     /// <exception cref="ArgumentNullException"><paramref name="messages"/> is <see langword="null"/>.</exception>
+    /// <remarks>
+    /// Applications must consider the risks of prompt injection attacks, data sizes, and the number of messages
+    /// sent to the underlying provider or returned from it. Unless a specific <see cref="IChatClient"/> implementation
+    /// explicitly documents safeguards for these concerns, the application is expected to implement appropriate protections.
+    /// </remarks>
```

Comment:
How about adding it as a paragraph to the remarks on the type rather than on each method?

Reply:
Great idea. Thank you.